### ISSUE TITLE
Allowing actor image to include extension in filename

### ISSUE THREAD

**Issue by penguintutor** [lordmauve/pgzero#135]
Whenever a new actor is created and the image is specified then the image must be specified without the extension (eg. without .bmp / .png). I understand this as a useful feature for those new to programming (particularly on Windows where the extension is often hidden), but this can cause confusion for those with a little more experience - perhaps those with programming experience but first time creating graphical programs / games.

The error message is not particularly useful giving the error:
KeyError: "No image found like '<filename>.png'. Are you sure the image exists?"

Especially when you look and see that the image does exist.

I would suggest upading the loader so it first tries without adding any extension before trying the popular extensions as it does currently. 

This could also be handled the same for sounds or any other files. 

Or alternatively at least update the error message (although I would prefer for the extension to just be optional)

I'm happy to have a go at implementing this and sending a PR if you think it would be useful.

**Comment by charlesej** [comment]
I've tried the following code to reproduce this issue, but as long as `images/alien.png` exists it seems to work.

```
alien = Actor('alien.png')
alien.image = 'alien'
alien.image = 'alien.png'

alien = Actor('alien')
alien.image = 'alien.png'
alien.image = 'alien'
```

**System details:**
- windows 10
- python: 3.5.0
- pygame: 1.9.4
- pgzero: 1.2 (also tried master branch at cfa8f6ed119d8959451a0b72be63cdd7a8b29489)
- numpy: 1.15.4

Could you provide some more details on how to reproduce this issue?
Thanks.

**Comment by penguintutor** [comment]
It looks like this may have already been dealt with. Perhaps under commit 6ea0280 (issue 101 - which is a different but related feature request).

I'm still seeing this problem when testing on a Raspberry Pi which has an older version of pgzero, but when I try it on a more recent version then it is all working as it should. 

It looks like the fix was released just before I raised this issue, but I was running a version that I had installed before the fix.

Thanks for looking into this, it looks like it's already fixed. - Setting the request to closed.
